packages/venia-ui/lib/components/MiniCart/miniCart.js

```diff
@@ -2,10 +2,19 @@ import React from 'react';
 import { bool, shape, string } from 'prop-types';
 
 import { useScrollLock } from '@magento/peregrine';
+import { useMiniCart } from '@magento/peregrine/lib/talons/MiniCart/useMiniCart';
 import { mergeClasses } from '@magento/venia-ui/lib/classify';
 
+import ProductList from './ProductList';
+
+import MiniCartOperations from './miniCart.gql';
+
 import defaultClasses from './miniCart.css';
 
+const Error = () => {
+    return <div>TBD</div>;
```

Comment:
You're gonna handle this right?

Reply:
I asked a question on the top of this PR, may be lost in the conversation. What do you guys think should happen in case of an error? Show error toast or the GQL message in the contents section?

Reply:
A toast is a good starting point. @tjwiebell did some work recently to handle errors, you may want to check out his pr: https://github.com/magento/pwa-studio/pull/2495/files

Reply:
Handled in a811662